runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -478,4 +487,9 @@ private boolean isInKraken()
 		return ArrayUtils.contains(client.getMapRegions(), KRAKEN_REGION);
 	}
 
+	private boolean isInKBD()
+	{
+		return ArrayUtils.contains(client.getMapRegions(), KBD_REGION);
```

Comment:
Put a local player plane check here (plane should be 0).

Reply:
Why do we do this? Should we also put the check in the IsInKraken method too, or is it a different case?

Reply:
NMZ uses the same map region at the KBD lair. The way we differentiate between the two is is the plane.

This is of course assuming NMZ instances use normal instance logic when it comes to dropped and looted items - definitely worth checking. If they use the same logic as a private KBD instance then of course you don't need to add a plane check but would want to rename the method to isInKBDorNMZ or something akin to that. If they use alternative logic to regular instances and private KBD instances then you'll want to add a new isInNMZ method (using the same map region but different plane check) and implement the new despawn logic similar to how you did for KBD.

Reply:
Ah, I see that makes sense.

So I tested out how loot and dropped items work in NMZ and it's pretty confusing...
From what I can tell items dropped in NMZ never despawn. I waited for over an hour but they didn't despawn.
Loot in NMZ uniformly despawn after 3 minutes which is what you would expect, but the weird thing is RuneLite only seems to be detecting some of the dropped object items for timers. It doesn't show timers for any of the bones/ashes dropped by monsters, but it does show timers for some of the other dropped items such as leaves dropped by Tanglefoot and vampyre dust dropped by Dessous. However, other drops have no timers such as choc-ice dropped by Karamel and cake dropped by the Culinromancer. I didn't test every boss, but I have no idea why some items have timers and some don't. Apart from that, it should be straightforward to add another if/else for NMZ's drop/loot mechanics too. I attached a image that shows some of what I'm talking about.
![image](https://user-images.githubusercontent.com/28718724/90721229-2e0ae500-e26d-11ea-9188-4243fd0f8c8b.png)

Reply:
Ok, so the reason the bones had to timers is because I was dumb and had them hidden by accident. I've updated it so dropped items in NMZ have no despawn timer (since they never despawn) and monster drops have the regular loot timer which is accurate based on my testing.